### PR TITLE
Added Cloning pod

### PR DESCRIPTION
**Pull request by MANDAL0R3**
Creates characters at a high cost of steel, and a long build and work
time of 10 seconds.
Once job is complete, the cloning pod is immediately destroyed(maybe
just damaged).
NOTE: after 4 characters building together, lag starts to occur.

screenshot1
screenshot2

Here is the link to my last pull request before i figured reverted and removed the resource files changes.
https://github.com/TeamPorcupine/ProjectPorcupine/pull/243

### CODE REVIEW

**Comment by henninglive** [comment]
Looks great, maybe add link in description to previous pull request so people can discussion. The problem with the font files is a known problem and should be fixed by #187.

**Comment by ComputerOverlord** [comment]
Should it also require power?

File: Assets/Scripts/Models/World.cs [hunk 1]
```diff
@@ -156,10 +156,14 @@ void SetupWorld(int width, int height)
...
-        foreach (Character c in characters)
+        for (int i = 0; i < characters.Count; i++)
         {
-            c.Update(deltaTime);
+            characters[i].Update(deltaTime);
         }
+        /*foreach (Character c in characters)
+        {
+            c.Update(deltaTime);
+        }*/
```
Comment (vogonistic) [inline Assets/Scripts/Models/World.cs]: How come this was changed? If there is a good reason to change it, please delete the commented out section.

**Comment by vogonistic** [comment]
If you take care of the comment I had on World.cs and re-indent the xml and lua, I'll merge this in the morning.

**Comment by MANDAL0R3** [comment]
OK I fixed it up but im not entirely sure whats up with the indentation. I just looked at what it shows in Files Changed but that's not what it looks like on my side.

**Comment by MANDAL0R3** [comment]
Thanks for letting me know about the indentation on the xml. I'm a little confused on how i did not notice it.

**Comment by quill18** [comment]
Seems to be in a good state now, to me. Merging!

**Pull request merged**